packages/embed-jsx/package.json

```
@@ -0,0 +1,47 @@
+{
+	"name": "@sapphire/embed-jsx",
+	"version": "0.0.1",
+	"description": "An easy way to use JSX to create discord embeds with discord.js",
+	"keywords": [
+		"embed",
+		"discord",
+		"bot",
+		"typescript",
+		"ts",
+		"yarn",
+		"sapphire"
+	],
+	"author": "@sapphire",
+	"homepage": "https://github.com/sapphire-project/utilities/tree/main/packages/embed-jsx",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"directories": {
+		"lib": "lib",
+		"test": "__tests__"
+	},
+	"files": [
+		"dist",
+		"!dist/.tsbuildinfo"
+	],
+	"engines": {
+		"node": ">=12",
+		"npm": ">=6"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/sapphire-project/utilities.git",
+		"directory": "packages/embed-jsx"
+	},
+	"scripts": {
+		"build": "tsc -p ./src"
+	},
+	"bugs": {
+		"url": "https://github.com/sapphire-project/embed-jsx/issues"
+	},
+	"peerDependencies": {
+		"discord.js": "^12.5.3"
```

```suggestion
		"discord.js": "^12.5.3"
```
Unless this particular version adds something required in the util, I'd suggest making it `12.x`.

---

Older versions are discouraged as they not longer work.

---

any discord.js version that has a MessageEmbed structure that matches the current one should work fine, as that's the only import